README.md

```diff
@@ -6,13 +6,58 @@
 
 # electron-starter
 
-This is a electron starter kit from some random guy on the Internet.
+This is a `TypeScript` and `React` starter kit for building Electron apps.
 
-[Check out the docs](https://skellock.github.io/electron-starter) for all the gory details.
+[Check out the docs](https://skellock.github.io/electron-starter) for some of the choices and motivations.
 
-# example
+
+### Example
+
+You have to bring your own awesome.  But here's a picture* after `npm i` and `npm start` and `npm storybook`...
 
 <img src='./docs/demo.gif' width='450' />
 
-It's not much to look at really. Just a barebones boilerplate. You need to bring your own awesome.
+><small>* What it looks like moments before you hit the delete button to nuke all the staged features..</small>
+
+
+# Out of the Box
+
+### Electron Features 💫
+
+* ability to build all 3 platforms on your mac
```

Comment:
Why call out mac specifically? Can Windows not build a mac binary with Electron? Asking because I truly don't know.